schema/opentelemetry_configuration.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$id": "https://opentelemetry.io/otelconfig/OpenTelemetryConfiguration",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Is this just using the latest or is there something necessary in 2020-12 that is known to be needed eventually?

Curious since existing Erlang json schema lib only supports up to draft 6, so choosing the minimum required version would be nice, otherwise we have to add support for the latest for potentially something that works under an older draft.

Reply:
Here are the release notes for the versions that have been released after draft 6:

- [Draft 07](https://json-schema.org/draft-07/json-schema-release-notes.html)
- [2019-09](https://json-schema.org/draft/2019-09/release-notes.html)
- [2020-12](https://json-schema.org/draft/2020-12/release-notes.html)

The versions appear to not always be backwards compatible, so an implementation that supports 2020-12 wouldn't necessarily support Draft 06 as well. Somebody fact check me on this if you know better. 

If this turns out to be true, then just as erlang only supports up to draft 6, we'll encounter other implementations that don't support versions less than 2020-12. 

 Not sure which features of JSON Schema we'll want to end up using. However, while I can't pin down exactly when draft 6 was released, its fair to say its quite old and I'm inclined to use the most recent stable version. It looks like there is an effort [stabilize JSON Schema](https://json-schema.org/blog/posts/future-of-json-schema) in the next release, which would help alleviate the problem since there would be a strong incentive for implementations to update to the stable version.

Reply:
Ah ok, if there isn't backwards compatibility (and I bet that is true since they are drafts) then trying to pin down a "minimum version" doesn't make sense.

Hopefully stabilization happens soon :)